src/query/execute_statement.c

```diff
@@ -18056,3 +18056,114 @@ do_create_server (PARSER_CONTEXT * parser, PT_NODE * statement)
 
   return error;
 }
+
+int
+get_dblink_info_from_dbserver (PARSER_CONTEXT * parser, const char *server, DB_VALUE * out_val)
```

Comment:
error-handling routine is not clear for me. It would better that either return the error directly or return it with a goto statement.

Reply:
It is unified by using the goto statement without separating the functions.